cmd/abapEnvironmentPushATCSystemConfig.go

```diff
@@ -62,6 +62,11 @@ func runAbapEnvironmentPushATCSystemConfig(config *abapEnvironmentPushATCSystemC
 	}
 	client.SetOptions(clientOptions)
 
+	connectionDetails.XCsrfToken, err = fetchXcsrfTokenFromHead(connectionDetails, client)
+	if err != nil {
+		return err
+	}
```

Comment:
maybe could be combined, e.g. if conne.. ; err != nil {...
```suggestion
	if connectionDetails.XCsrfToken, err = fetchXcsrfTokenFromHead(connectionDetails, client); err != nil {
		return err
	}
```